### PR TITLE
Desactivate named servers

### LINKED ISSUE #179: Default Plasma config should not use Named servers
Jupyer Hub [named servers](https://jupyterhub.readthedocs.io/en/stable/reference/config-user-env.html#named-servers) are very powerful but could be puzzling for users.

Plasma default should stick to only one default server. Of course, named servers can be activated later.

This new default will dramatically simplify *Hub Control Panel* going from:

image

to only:

image

Issue #130 could be reactivated once we will be able to simplify the Named servers UI and maybe have only servers named after environment names.

### PR DESCRIPTION
**Pull request by pierrepo**
- [x] Add / update the documentation

Fix #179

### CODE REVIEW

**Comment by pierrepo** [comment]
@jtpio I hope this PR is faithful to our discussion.

I tried to keep the documentation as concise as possible.

File: docs/configuration/namedservers.rst [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+   c.JupyterHub.allow_named_servers = True
+   c.JupyterHub.named_server_limit_per_user = 2
+
+Then, reload tljh:
+
+.. code-block:: text
+
+   sudo tljh-config reload
+
+Have a look to `named servers documentation <https://jupyterhub.readthedocs.io/en/stable/reference/config-user-env.html#named-servers>`_
```
Comment (jtpio) [inline docs/configuration/namedservers.rst:23]:
```suggestion
Have a look at the `named servers documentation <https://jupyterhub.readthedocs.io/en/stable/reference/config-user-env.html#named-servers>`_
```

File: tljh-plasma/tljh_plasma/__init__.py [hunk 2]
```diff
@@ -83,8 +83,7 @@ def tljh_custom_jupyterhub_config(c, tljh_config_file=CONFIG_FILE):
     c.JupyterHub.cleanup_servers = False
     c.JupyterHub.authenticator_class = PAMAuthenticator
     c.JupyterHub.spawner_class = PlasmaSpawner
-    c.JupyterHub.allow_named_servers = True
-    c.JupyterHub.named_server_limit_per_user = 2
+    c.JupyterHub.allow_named_servers = False
```
Comment (jtpio) [inline tljh-plasma/tljh_plasma/__init__.py:86]: We should be able to remove that line completely, as `allow_named_servers` default to `False`.

**Comment by jtpio** [comment]
Looks good, thanks Pierre :+1:

File: tljh-plasma/tljh_plasma/__init__.py (hunk 2 above)
Comment (pierrepo) [reply tljh-plasma/tljh_plasma/__init__.py:86]: You're perfectly right! Here's an excerpt from JupyterHub documentation:
>   allow_named_servers c.JupyterHub.allow_named_servers = Bool(False)

Ref: https://jupyterhub.readthedocs.io/en/stable/api/app.html

Comment (pierrepo) [reply tljh-plasma/tljh_plasma/__init__.py]: I'll create a commit to fix this.

**Pull request merged**